isup/isup-impl/src/main/java/org/mobicents/protocols/ss7/isup/util/BcdHelper.java

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright 2016 Grzegorz Figiel
+ *
```

Comment:
this license header is wrong and can't be accepted. Please use the standard TeleStax  license header from https://docs.google.com/document/d/1RZz2nd2ivCK_rg1vKX9ansgNF6NpK_PZl81GxZ2MSnM/edit#heading=h.iayg4mgzyzs8 for any changes done to any of the files modified that don't contain this header or any new files contributed